docs/architecture/Parser.mdx

```diff
@@ -42,7 +42,7 @@ interface ParserOptions {
   lib?: string[];
   moduleResolver?: string;
   program?: import('typescript').Program;
-  project?: string | string[];
+  project?: string | string[] | true;
```

Comment:
My first instinct was a -1 because the string form is simpler to read and I think clearer for newcomers... but now that the vast majority of projects is using `project: true` I think you're right. This makes sense.

Reply:
Actually, hmm. Now I'm thinking again. Will ruminate.

Reply:
Ok yes I'm back in camp `string | string[] | true`:
* Single string options are easier to read & simpler to configure with. Which is an optimization I think we should strive for - folks are sensitive to config complexity!
* It'd be a breaking change to _remove_ the option for `string`, and I'd love to get this into `main` before v6
Thoughts @bradzacher?

Reply:
We could do the removal separately, for sure.
The reason I think it's a good change is that unless you're in a really small repo - you're going to have 2 or more tsconfigs.
I think the majority of users probably use the array form - so offering both seems like overhead for us which increases complexity.